backend/bang-ggood/src/main/java/com/bang_ggood/global/exception/ExceptionCode.java

```diff
@@ -1,103 +1,101 @@
 package com.bang_ggood.global.exception;
 
+import lombok.Getter;
 import org.springframework.http.HttpStatus;
 
+@Getter
 public enum ExceptionCode {
 
     // 전체
-    INVALID_PARAMETER(HttpStatus.BAD_REQUEST, "잘못된 인자입니다."),
+    INTERNAL_SERVER_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.INTERNAL_SERVER_ERROR, "예상치 못한 서버에러가 발생했습니다"),
+    INVALID_PARAMETER(HttpStatus.BAD_REQUEST, BangggoodCode.INTERNAL_SERVER_ERROR, "잘못된 인자입니다."),
 
     // Option
-    OPTION_INVALID(HttpStatus.BAD_REQUEST, "잘못된 옵션 ID입니다."),
-    OPTION_DUPLICATED(HttpStatus.BAD_REQUEST, "중복된 옵션이 존재합니다."),
+    OPTION_INVALID(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "잘못된 옵션 ID입니다."),
+    OPTION_DUPLICATED(HttpStatus.BAD_REQUEST, BangggoodCode.CHECKLIST_ERROR, "중복된 옵션이 존재합니다."),
 
     // Question
-    QUESTION_ID_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "중복된 질문 ID가 존재해 질문을 생성할 수 없습니다."),
-    QUESTION_HIGHLIGHT_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, "잘못된 하이라이트 키워드가 존재해 질문을 생성할 수 없습니다."),
-    QUESTION_INVALID(HttpStatus.BAD_REQUEST, "잘못된 질문 ID입니다."),
-    QUESTION_DUPLICATED(HttpStatus.BAD_REQUEST, "중복된 질문이 존재합니다."),
-    QUESTION_DIFFERENT(HttpStatus.BAD_REQUEST, "수정할 질문 목록이 기존의 질문 목록과 동일하지 않습니다."),
+    QUESTION_ID_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.QUESTION_ERROR, "중복된 질문 ID가 존재해 질문을 생성할 수 없습니다."),
+    QUESTION_HIGHLIGHT_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.QUESTION_ERROR, "잘못된 하이라이트 키워드가 존재해 질문을 생성할 수 없습니다."),
```

Comment:
```suggestion
    QUESTION_HIGHLIGHT_ERROR(HttpStatus.INTERNAL_SERVER_ERROR, BangggoodCode.CHECKLIST_SERVER_ERROR, "잘못된 하이라이트 키워드가 존재해 질문을 생성할 수 없습니다."),
```
![image](https://github.com/user-attachments/assets/1c7037b8-1fdc-461b-a4bc-fceefa1a7fe0)